cmd/podman/shared/create.go

```diff
@@ -23,6 +23,7 @@ import (
 	ns "github.com/containers/libpod/pkg/namespaces"
 	"github.com/containers/libpod/pkg/rootless"
 	cc "github.com/containers/libpod/pkg/spec"
+	"github.com/containers/libpod/pkg/seccomp"
```

Comment:
Does it make sense to move this under specgen as a subpackage? I'm getting concerned about the number of things we have in pkg/

Reply:
I think it makes sense in `pkg/` while other packages in `pkg/` might very well belong into a (not yet existing) `internal/` package for stuff we don't want to expose to external users.

Reply:
the pkg scrub should come after specgen is used in the cli and varlink is gone imho